devel/qt5pas/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake5 1.0
+
+name                qt5pas
+version             2.0.10-2
+revision            0
+categories          devel
+platforms           darwin
+license             GPL-2 LGPL-2
+maintainers         {@kamischi web.de:karl-michael.schindler} openmaintainer
+description         Pascal wrapper for Qt5
+
+long_description    The Free Pascal Qt5 binding allows Free Pascal to interface with the \
+                    C++ Library Qt. \
+                    \
+                    This binding may not cover the whole Qt5 framework but only the classes \
+                    needed by the Cross Platform Lazarus IDE to use Qt as a Widget set.
+
+homepage            https://wiki.freepascal.org/Qt5_Interface
+master_sites        sourceforge:lazarus
+distname            lazarus-${version}
+checksums           rmd160  1e9c89a34bbdf90bf4010e73424b53be117e8e74 \
+                    sha256  64d5626468dd24a3332b205f3abd0a581ab7de1b060a2d57e21864066cfd43b7 \
+                    size    69626076
+
+supported_archs     x86_64 arm64
+
+# Note: qt5-qtbase is 5.15 and for macOS 10.13 and up, for earlier (10.7 - 10.12) use 5.6
```

Comment:
the qmake5 PortGroup takes care of including the qt5 PortGroup, and the qt5 PortGroup takes care of selecting and depends_lib'ing all the parts of qt5 that are needed.

Short story: this whole block is not likely needed, and potentially causes trouble.